graph/client.py

```diff
@@ -89,6 +89,9 @@ def parse_arguments(self, args):
         parser = argparse.ArgumentParser(
             description='Client for visualization of SCAP rule evaluation results')
 
+        parser.add_argument('--showFailRules', action="store_true", default=False, help='Show all FAIL rules')
```

Comment:
I think using camel cases is unusual in command line options, I guess it will confuse users. I recommend using `--show-failed-rules`. Similarly for all the remaining command line options.

Reply:
The help doesn't explain if it shows only failed rules (I suppose).